web/src/main/java/org/cbioportal/web/mixin/MutationSpectrumMixin.java

```diff
@@ -0,0 +1,19 @@
+package org.cbioportal.web.mixin;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class MutationSpectrumMixin {
+
+    @JsonProperty("CtoA")
+    private Integer ctoA;
```

Comment:
I needed this mixin because of a bug in Jackson, which was fixed in a later version but I didn't want to update the version yet. And `CtoA` vs `cToA` is a personal preference, I find the first one more clear.